Resources/Prototypes/_RMC14/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -1,11 +1,29 @@
 - type: entity
-  parent: BaseWindoor
-  id: CMWindoor
-  name: glass door
-  description: A window, that is also a door. A windoor if you will.
+  id: CMBaseWindoor
+  parent: BaseStructure
```

Comment:
Parent should go above id per the style rules https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html?highlight=parent#entities